docs/source/API/core/policies/NestedPolicies.rst

```diff
@@ -0,0 +1,121 @@
+``NestedPolicies``
+==================
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+``Kokkos::PerTeam``
+-------------------
+
+``Kokkos::PerThread``
+---------------------
+
+``Kokkos::TeamThreadRange``
+---------------------------
+
+``Kokkos::TeamThreadMDRange``
+-----------------------------
+
+``Kokkos::TeamVectorRange``
+---------------------------
+
+``Kokkos::TeamVectorMDRange``
+-----------------------------
+
+``Kokkos::ThreadVectorRange``
+-----------------------------
+
+``Kokkos::ThreadVectorMDRange``
+-------------------------------
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+~~~~~
+
+.. code-block:: cppkokkos
```

Comment:
code block cannot use cppkokkos, a code block is not a sphinx thing it is a rst thing.
Please use 
```
.. code-block:: cpp
```